src/main/java/org/simulator/examples/SBMLTestSuiteRunnerWrapper.java

```diff
@@ -50,10 +48,10 @@ public static void main(String[] args) throws IOException, XMLStreamException, M
         String level = args[3];
         String  version = args[4];
 
-        String filePath = dirPath + '/' + currentCase + '/' + currentCase + "-sbml-l" + level + 'v' + version + ".xml";
-        String settingsPath = dirPath + '/' + currentCase + '/' + currentCase + "-settings.txt";
-        String outputFilePath = outputDirPath + '/' + currentCase + ".csv";
-        String resultsPath = dirPath + '/' + currentCase + '/' + currentCase + "-results.csv";
+        String filePath = dirPath + File.separator + currentCase + File.separator + currentCase + "-sbml-l" + level + 'v' + version + ".xml";
+        String settingsPath = dirPath + File.separator + currentCase + File.separator + currentCase + "-settings.txt";
+        String outputFilePath = outputDirPath + File.separator + currentCase + ".csv";
+        String resultsPath = dirPath + File.separator + currentCase + File.separator + currentCase + "-results.csv";
```

Comment:
Did anybody test this under Windows? To my knowledge, internally Java always uses `/` as separator in paths and the virtual machine translates it to the operating system.

Reply:
Just to add: I think this will work, but I also think it is not needed (without having it tested).

Reply:
Yes, I just saw that when we give `/`, then Java internally handles things to translate to platform-specific code. But `File.separator` can be a good way to show users what will make sense in their OS, rather than what makes sense to Java.